cppcache/integration/framework/Cluster.h

```diff
@@ -127,6 +127,8 @@ using ConserveSockets = NamedType<bool, struct useConserveSocketsParameter>;
 
 class Cluster {
  public:
+  enum class Subscription_State { Enabled, Disabled };
```

Comment:
This doesn't match the naming conventions used in the framework.
I love the introduction of a named type to replace a general purpose type!!!

Reply:
Thy will be done.